backends/ida/graffiti.py

```diff
@@ -235,7 +238,28 @@ def activate(self, ctx):
     def update(self, ctx):
         return idaapi.AST_ENABLE_ALWAYS
 
-def sync_read_thread():
+
+def bring_ida_to_foreground():
+    # according to https://hex-rays.com/blog/plugin-focus-heimdallr/
+    # Tested only on macos, should be adapted easily to Linux/Windows
+    if sys.platform == 'darwin':
```

Comment:
Support windows part 1
```suggestion
    if sys.platform in ['darwin', 'win32]:
```